src/rp2_common/hardware_xosc/xosc.c

```diff
@@ -13,14 +13,30 @@
 #include "hardware/regs/xosc.h"
 #include "hardware/structs/xosc.h"
 
+// Allow lengthening startup delay to accommodate slow-starting oscillators
+
+// PICO_CONFIG: PICO_XOSC_STARTUP_DELAY_MULTIPLIER, Multiplier to lengthen xosc startup delay to accommodate slow-starting oscillators, type=int, min=1, default=1, group=hardware_xosc
```

Comment:
stylistically (going forward, there may be a few stragglers) we are trying to put these  defaulted values in the header in case any other code (source files) cares about the actual value. can you move this to xsoc.h

Reply:
Sure. Moved PICO_CONFIG and default setting to `src/rp2_common/hardware_xosc/include/hardware/xosc.h`, and changed `#include` in xosc.c to accommodate this.